src/CommandLineArgumentParser.c

```diff
@@ -131,7 +131,7 @@ RTI_Retval CommandLineArgumentParser_parse_arguments(
 
                     if (Common_allocate_string(
                                 &output_values->qos_library, 
-                                string_size) != RTI_OK) {
+                                string_size) != DDS_BOOLEAN_TRUE) {
```

Comment:
don't compare with DDS_BOOLEAN_TRUE/FALSE. Since it's a boolean, use !

Reply:
Done.